src/backend/chat.nim

```diff
@@ -75,15 +77,18 @@ proc sendChatMessage*(
     }
   ])
 
-proc sendImages*(chatId: string, images: var seq[string]): RpcResponse[JsonNode] {.raises: [Exception].} =
+proc sendImages*(chatId: string, images: var seq[string], msg: string): RpcResponse[JsonNode] {.raises: [Exception].} =
+  # Add "placeholder" text to instead of empty string as mobile app
+  let text = if (msg == ""): EMPTY_IMAGE_MESSAGE else: msg
```

Comment:
This should be done in the `service`. That way, we don't need to `EMPTY_IMAGE_MESSAGE` in two places